api/overview/azure/search.md

```diff
@@ -14,21 +14,25 @@ ms.service: search
 
 ## Libraries for data access
 
-The `Azure.Search.Documents` client library (v1) is a new offering for .NET developers who want to use search technology in their applications. This version supersedes the `Microsoft.Azure.Search` client library (v10) with full functional parity.
+After an Azure Cognitive Search resource is created and configured, use data access libraries to create and consume search objects in client applications.
 
-The APIs are similar but not identical. When reviewing code samples and content, be sure to check for the namespace (`using Azure.Search.Documents;`) to confirm which client ibrary is in use.
+### Version 11
 
-### Version 1.x
+The **`Azure.Search.Documents`** is a new client library  for .NET developers who want to use search technology in their applications. This version is functionally equivalent to the v10 legacy client library, but takes dependencies on **`Azure.Core`** and **`System.Text.Json`** to provide a standard approach for service configuration, authentication, document serialization, and other tasks.

```

Comment:
The version that's available right now isn't actually "functionally equivalent" yet, but the August release should be. Do we want to delay publishing this a bit, or put a footnote here or something?

CC @tg-msft @heaths for their opinions

Reply:
Neither v10 nor v11 have knowledge store, but they do have skillsets.  Both target 2019-05-06 REST API.  In that regard, I see them as functionally equivalent.  In August, v11 will be ahead when it adds kstore support (and thus targets 2020-06-30).  I can change the wording then?